homeassistant/components/evse_wifi/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""The EVSE Wifi integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_HOST, CONF_INTERVAL, CONF_MAX_CURRENT, CONF_NAME, DOMAIN
+from .evse import EVSE
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+    Platform.NUMBER,
+    Platform.BUTTON,
+    Platform.SWITCH,
+]
```

Comment:
Only do one platform for each PR. Pick one of these you want to start with and use that

Reply:
I'll just review sensor as it is probably the most fleshed out one/the one you will want first